step1/vanilla-todo/js/Controller.js

```diff
@@ -0,0 +1,22 @@
+export default class Controller {
+  constructor(store, { todoInputFormView, todoListView }) {
+    this.store = store;
+    this.todoInputFormView = todoInputFormView;
+    this.todoListView = todoListView;
+    this.subscribeViewEvents();
+
+    this.render();
+  }
```

Comment:
```suggestion
export default class Controller {
  store;
  todoInnputFormView;
  todoListView;
  
  constructor(store, { todoInputFormView, todoListView }) {
    this.store = store;
    this.todoInputFormView = todoInputFormView;
    this.todoListView = todoListView;
    this.subscribeViewEvents();

    this.render();
  }
```
이렇게 어떤 멤버변수를 사용하는지 클래스 필드에 미리 정의해주면 좋답니다!